### DISCUSSION TITLE
Problem with embedding video in a page

### DISCUSSION THREAD

**Discussion by marcmaurer99** [requarks/wiki#4571] [Help / Questions]
Hello,

I created a page where I wanted to embed a youtube video. It worked in the edit mode and I can see that the video is embedded. Then I save the page but afterward the embedded video is no longer visible in the non-edit mode. If I enter edit mode again then I can see the embedded video. I did turn on the iframe option in the preferences. Is there something else I need to change?

Thanks for your help,

Marc

**Comment by majianjia** [comment]
I also enconter this problem.
After getting into the source html, I found that the y size of iframe is set to zero.

![image](https://user-images.githubusercontent.com/3821068/178715015-4b123d7a-6cf2-4be3-b873-92b15903f73c.png)
![image](https://user-images.githubusercontent.com/3821068/178715499-2f048ac2-7516-471d-995c-77efc9f53fdb.png)

**Comment by majianjia** [comment]
solution 
https://github.com/requarks/wiki/discussions/5043
